client/src/components/Portfolio/helpers/getPortfolioValue.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+const getPortfolioValue = (user) => {
+  const stocks = user.userPortfolio;
+  const stockData = stocks.slice();
+  user.portfolioValue = 0;
+
+  return new Promise((resolve, reject) => {
+    axios.all(stockData.map((stock) =>
+      axios.get('/fetchSelectedStock', {
```

Comment:
This is the call that will hit our API multiple times to get the current value of stock, right?  Thanks for putting this together!

Reply:
Correct!!